sdk/iot/Azure.Iot.Hub.Service/src/swagger/iothubservice.json

```diff
@@ -1,3093 +1,3258 @@
+                    "application/json"
+                ],
+                "parameters": [
+                    {
+                        "name": "digitalTwinId",
+                        "in": "path",
+                        "required": true,
+                        "type": "string"
+                    },
+                    {
+                        "name": "interfaceName",
+                        "in": "path",
+                        "required": true,
+                        "type": "string"
+                    },
+                    {
+                        "name": "commandName",
+                        "in": "path",
+                        "required": true,
+                        "type": "string"
+                    },
+                    {
+                        "name": "payload",
+                        "in": "body",
+                        "required": true,
+                        "schema": {
+                            "description": "The Request Payload",
+                            "type": "object"
+                        }
+                    },
+                    {
+                        "$ref": "#/parameters/api-version"
+                    },
+                    {
+                        "name": "connectTimeoutInSeconds",
+                        "in": "query",
+                        "description": "Connect timeout in seconds.",
+                        "required": false,
+                        "type": "integer",
+                        "format": "int32"
+                    },
+                    {
+                        "name": "responseTimeoutInSeconds",
+                        "in": "query",
+                        "description": "Response timeout in seconds.",
+                        "required": false,
+                        "type": "integer",
+                        "format": "int32"
+                    }
+                ],
+                "responses": {
+                    "200": {
+                        "description": "Returns the digital twin command response payload",
+                        "schema": {
+                            "description": "The Response Payload",
+                            "type": "object"
+                        },
+                        "headers": {
+                            "x-ms-command-statuscode": {
+                                "description": "Device Generated Status Code for this Operation",
+                                "type": "integer",
+                                "format": "int32"
+                            },
+                            "x-ms-request-id": {
+                                "description": "Server Generated Request Id (GUID), to uniquely identify this request in the service",
+                                "type": "string"
+                            }
+                        }
+                    }
+                }
+            }
+        },
+        "/devices/{deviceId}/message": {
```

Comment:
Service has this in the format: `/messages/deviceBound`. This is our recommendation.

Reply:
[msdn docs](https://docs.microsoft.com/en-us/azure/iot-hub/iot-hub-devguide-messages-c2) has this documented as: 
You send cloud-to-device messages through a service-facing endpoint, /messages/devicebound. A device then receives the messages through a device-specific endpoint, /devices/{deviceId}/messages/devicebound.

So I am not confident that we'll be able to modify this.